c/driver/postgresql/connection.cc

```diff
@@ -72,12 +72,44 @@ class PqResultRow {
 // as expected prior to iterating
 class PqResultHelper {
  public:
-  PqResultHelper(PGconn* conn, const char* query) : conn_(conn) {
+  PqResultHelper(PGconn* conn, const char* query, std::vector<std::string> param_values,
```

Comment:
```suggestion
  explicit PqResultHelper(PGconn* conn, const char* query, std::vector<std::string> param_values,
```

Should've caught this earlier but might as well now